app/Http/Controllers/Backoffice/Sales/Clients/ClientPostController.php

```
@@ -0,0 +1,134 @@
+             *  name: string,
+             *  description: string|null
+             * } $validatedData
+             */
+            $validatedData = $request->validate([
+                'id' => 'required|string|uuid',
+                'name' => 'required|string|min:3|max:40',
+                'description' => 'nullable|string|min:3|max:100'
+            ]);
+
+            /** @var string $tenantId */
+            $tenatId = tenant('id');
```

PHPStan (2.0.3)

Variable $tenantId in PHPDoc tag @var does not match assigned variable $tenatId.


(varTag.differentVariable)